concepts/observability-strategy/strategy.md

```diff
@@ -57,12 +63,12 @@ To sum it up, the goals of Kyma observability should be:
 
 ## Architecture
 
-The proposal introduces a new preconfigured agent layer that's responsible for collecting all telemetry data. Hereby, the collection is very dependent on the signal type. The agents can be configured at runtime with different signal pipelines with basic filtering (inclusion and exclusion of signals) and outputs, so that the agents start shipping the signals through the pipelines to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core component called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. The users will still be able to install them manually with a blueprint, but they will no longer be part of the Kyma offering.
+A new preconfigured agent layer gets introduced which is responsible for collecting all telemetry data without being bound to any backend. Hereby, the collection is very dependent on the signal type. The agents can be configured at runtime with different signal pipelines with basic filtering (inclusion and exclusion of signals) and outputs, so that the agents start shipping the signals through the pipelines to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core component called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. The users will still be able to install them manually with a blueprint, but they will no longer be part of the Kyma offering.
 
-![b](./assets/future.drawio.svg)
+![b](./assets/strategy-future.drawio.svg)
 
-As mentioned before, the technology and protocols for the signal collection are different for the signal types: Logs are tailed from container log files, metrics are pulled using the Prometheus format usually, and traces are pushed with OTLP. With that, also the pre-integration (so that typical signals are collected instantly) is different per type.
-That's why the specific concepts for the different types are different, and are discussed in more detail in the individual concepts:
+As mentioned before, the technology and protocols for the signal collection are different for the signal types: Logs are tailed from container log files, metrics usually are pulled using the Prometheus format, and traces are pushed with OTLP. With that, also the pre-integration (so that typical signals are collected instantly) is different per type.
```

Comment:
```suggestion
As mentioned before, the technology and protocols for the signal collection depend on the signal types: Logs are tailed from container log files, metrics usually are pulled using the Prometheus format, and traces are pushed with OTLP. With that, also the pre-integration (so that typical signals are collected instantly) is different per type.
```